pkgs/development/python-modules/simpful/default.nix

```diff
@@ -22,7 +22,7 @@ buildPythonPackage rec {
     owner = "aresio";
     repo = pname;
     rev = "refs/tags/${version}";
-    hash = "sha256-1CU/Iz83CKRx7dsOTGfdJm98TUfc2kxCHKIEUXP36HQ=";
+    hash = "sha256-54WkKnPB3xA2CaOpmasqxgDoga3uAqoC1nOivytXmGY=";
   };
 
   # patch dated use of private matplotlib interface
```

Comment:
The fix for this should be tagged in this release

Reply:
Thanks, fixed.